scripts/lib/lib.js

```diff
@@ -1,4 +1,4 @@
-const {execSync} = require('child_process');
+const { execSync } = require("child_process");
```

Comment:
our linter usually wants strings with a single quote in JS. Why we are using double qoutes?

Reply:
I've normalised to single quotes: `npx prettier --single-quote scripts/*`